src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/Set-AzConnectedKubernetes.ps1

```diff
@@ -78,6 +78,33 @@ INPUTOBJECT <IConnectedCluster>:
 .Link
 https://learn.microsoft.com/powershell/module/az.connectedkubernetes/set-azconnectedkubernetes
 #>
+function Merge-MaybeNullInput {
+    [Microsoft.Azure.PowerShell.Cmdlets.ConnectedKubernetes.DoNotExportAttribute()]
+    param(
+        [Microsoft.Azure.PowerShell.Cmdlets.ConnectedKubernetes.Models.Api20240715Preview.IConnectedCluster]
+        $InputObject,
+        [System.Collections.Generic.Dictionary[system.String, System.Object]]
+        $LclPSBoundParameters
+    )
+
+    $mergeFields = @(
```

Comment:
Maybe worth a comment explain these are optional fields and we'll read value directly if provided

Reply:
Done.